From f3ff889a98965411d78c40826880619a1b34e400 Mon Sep 17 00:00:00 2001
From: Adrien Ricciardi <aricciardi@baylibre.com>
Date: Tue, 26 Sep 2023 11:50:29 +0200
Subject: [PATCH 09/38] machines: posix: Fixed an unused parameter warning in
 tpl_create_context_trampoline().

The 'sigid' arameter is not used and triggers a compilation warning.

Signed-off-by: Adrien Ricciardi <aricciardi@baylibre.com>
---
 machines/posix/tpl_posix_context.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/machines/posix/tpl_posix_context.c b/machines/posix/tpl_posix_context.c
index c961cdd3..28bd59ea 100644
--- a/machines/posix/tpl_posix_context.c
+++ b/machines/posix/tpl_posix_context.c
@@ -79,6 +79,8 @@ FUNC(void, OS_CODE) tpl_create_context_boot(void)
 #include "tpl_memmap.h"
 FUNC(void, OS_CODE) tpl_create_context_trampoline(int sigid)
 {
+    (void) sigid;
+
     /* 5 : new context created. We go back to tpl_init_context */
     if( 0==setjmp(tpl_stat_proc_table[new_proc_id]->context->initial) )
     {
-- 
2.34.1

